testing/MLDB-1336-builtin-checks.py

```diff
@@ -0,0 +1,21 @@
+
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+
+mldb = mldb_wrapper.wrap(mldb) # noqa
+
+import unittest
+
+class myTest(unittest.TestCase):
+
+    def test_sequence(self):
+        with self.assertRaisesRegexp(mldb_wrapper.ResponseException, "Executing builtin function exp: Can't convert value 'a' to double") as re:
```

Comment:
Arg actually you were right, I can't do this without adding a try_catch to all builtins, or doing some complex registering of expected arguments types. Is it worth the effort at this point?